ci/docker/python-wheel-manylinux.dockerfile

```diff
@@ -75,11 +78,15 @@ COPY ci/scripts/install_vcpkg.sh \
 ENV VCPKG_ROOT=/opt/vcpkg
 ARG build_type=release
 ENV CMAKE_BUILD_TYPE=${build_type} \
-    VCPKG_FORCE_SYSTEM_BINARIES=1 \
-    VCPKG_OVERLAY_TRIPLETS=/arrow/ci/vcpkg \
+    GITHUB_REPOSITORY_OWNER="${GITHUB_REPOSITORY_OWNER}" \
+    GITHUB_TOKEN="${GITHUB_TOKEN}" \
+    VCPKG_BINARY_SOURCES="${VCPKG_BINARY_SOURCES}" \
     VCPKG_DEFAULT_TRIPLET=${arch_short}-linux-static-${build_type} \
-    VCPKG_FEATURE_FLAGS="manifests"
+    VCPKG_FEATURE_FLAGS="manifests" \
+    VCPKG_FORCE_SYSTEM_BINARIES=1 \
+    VCPKG_OVERLAY_TRIPLETS=/arrow/ci/vcpkg
 
+# TODO: Use --mount=type=secret for GITHUB_TOKEN
```

Comment:
Is there an issue open for this? If so, can we mention the GH issue number?

Reply:
No. I'll open it when this is merged or someone agrees that we should do it.